frontend/src/app/services/desk.service.ts

```diff
@@ -0,0 +1,34 @@
+import { Desk } from './../model/desk.model';
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+@Injectable({
+  providedIn: 'root'
+})
+
+@Injectable()
+export class DeskService {
+
+  url = 'http://localhost:64513/api/desks';
+  desk: Desk[];
+  tmp = [];
```

Comment:
zamiast korzystać z takiej tablicy `any` lepiej zrobić model na wyświetlenie kolejnych wierszy zakładam, że to nie będzie tylko wartość ale też etykieta, np. "Imię: Xyz"

Reply:
trochę tego nie widzę, możesz sprecyzować?

Reply:
Musisz zastanowić się jaki jest przepływ danych w aplikacji, bo w tym momencie, jest jeszcze problem, gdzie komponent biurka subskrybuje po dane z API które przekazuje do serwisu, które to trafiają do komponenty details. Ale problem jest taki, że dane powinniśmy pobrać na początku i potem API nie wykorzystujemy do momentu admin panelu.
Co do modelu opisałem to w konwersacji przy html.
Jak będziesz miał więcej pytań to pisz.